src/bundlingCalls/bundleExecutor.ts

```diff
@@ -60,7 +60,8 @@ export class BundleExecutor {
   _options: BundleOptions;
   _descriptor: BundleDescriptor;
   _tasks: {[index: string]: Task};
-  _timers: {[index: string]: NodeJS.Timer};
+  // _timers: {[index: string]: NodeJS.Timer };
```

Comment:
Is this comment necessary?

Reply:
Should be removed, I was working with that a long time ago but I guess I ultimately didn't change anything

Reply:
Yeah, I guess this file should not be changed at all.  `git checkout master -- src/bundlingCalls/bundlingExecutor.ts`